requirements.txt

```diff
@@ -24,9 +25,10 @@ fastapi==0.95.0
 uvicorn==0.21.1
 frozenlist==1.3.3
 google-api-core==2.11.0
-google-api-python-client==2.85.0
+google-api-python-client==2.90.0
```

Comment:
It looks like not used in our brain. It might be removed.
To: @jupiter0913

Reply:
I think it should be with v2.90.0 for email plugin